apps/dashboard/components/calendar-event/calendar-event.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components';
+import { H2 } from '@hibiscus/ui';
+import { Colors2023 } from '@hibiscus/styles';
+import { AiOutlineClockCircle } from 'react-icons/ai';
+import { MdLocationOn } from 'react-icons/Md';
+
+/* eslint-disable-next-line */
+export interface CalendarEventProps {}
+
+const Day = '04';
+const Month = 'Feb';
+const EventName = 'Puppy Pen';
+const EventLocation = 'TTH 102';
+const EventPoints = '100';
+const StartAndEndTime = '12:00 - 3:00 pm';
```

Comment:
we store `start` and `end` times as Javascript dates instead of them together like this